core/src/test/scala/integration/kafka/api/IntegrationTestHarness.scala

```diff
@@ -95,9 +95,12 @@ abstract class IntegrationTestHarness extends KafkaServerTestHarness {
 
   @After
   override def tearDown() {
-    producers.foreach(_.close())
-    consumers.foreach(_.close())
-    super.tearDown()
+    try {
+      producers.foreach(_.close())
+      consumers.foreach(_.close())
+    } finally {
+      super.tearDown()
+    }
```

Comment:
This is a bit tricky. Maybe we should just leave as it was. The issue is that `close` throwing an exception is a bug and if we do the try/finally, we may obscure the original exception if `super.tearDown` itself throws an exception.